src/opening-period/form/OpeningPeriodForm.tsx

```diff
@@ -76,26 +87,51 @@ export default function OpeningPeriodForm({
     openingPeriodBeginDate: datePeriod?.start_date,
     openingPeriodEndDate: datePeriod?.end_date,
     timeSpans: spans,
+    rules,
   };
 
-  const { register, handleSubmit, errors, control } = useForm<
+  const { register, handleSubmit, errors, control, setValue } = useForm<
     OpeningPeriodFormData
   >({
     mode: 'all',
     defaultValues: formValues,
   });
+
   const { fields, append, remove } = useFieldArray({
```

Comment:
I wonder if it would be good idea to rename these too for them to be consistent with how we name these in rows 107 - 109?

Reply:
I can change these as well

Reply:
- [x] changed